src/sql/platform/query/common/queryRunner.ts

```diff
@@ -347,11 +349,12 @@ export default class QueryRunner extends Disposable {
 			let hasShowPlan = !!result.resultSetSummary.columnInfo.find(e => e.columnName === 'Microsoft SQL Server 2005 XML Showplan');
 			if (hasShowPlan) {
 				this._isQueryPlan = true;
+
 				this.getQueryRows(0, 1, result.resultSetSummary.batchId, result.resultSetSummary.id).then(e => {
 					if (e.resultSubset.rows) {
 						this._planXml.resolve(e.resultSubset.rows[0][0].displayValue);
 					}
-				});
+				}).catch(this._logService.error);
```

Comment:
should it be 
```.catch((e) => this._logService.error(e))``` instead ?

Reply:
e gets implicitly passed in i think

Reply:
```
(async() => {
    throw new Error('boop');
})().catch(console.error);
```

yep

Reply:
In either case you are passing through a function so it doesn't really matter, but `.catch((e) => this._logService.error(e))` is a little more explicit and easier to understand.